### PR TITLE
added focus link styling for table row anchor links

### PR DESCRIPTION
**Pull request by phillip-whitaker**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-2031

by accident, it also resolves: https://tools.hmcts.net/jira/browse/EUI-2033

### Change description ###

Changed link text so it matches GDS style and satisfies WCAG AA

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/shared/components/search-result/search-result.component.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
 $blue_table: #005ea5;
 $blue_pagination: #005da6;
 $black: #231F20;
+$focus-yellow: #FFDD00;
```
Comment (paul-graham) [inline src/shared/components/search-result/search-result.component.scss:9]: Why not use `$govuk-focus-colour` as it's already part of the GDS colour palette?

File: src/shared/components/search-result/search-result.component.scss [hunk 2]
```diff
@@ -16,9 +17,19 @@ table {
   tbody tr td {
     font-size: 16px;
     word-wrap: break-word;
+
     a {
+      float: left;
       color: $blue_table;
       background-color: $white;
+
+      &.govuk-link:focus {
```
Comment (paul-graham) [inline src/shared/components/search-result/search-result.component.scss:26]: What are we doing to links that means the `govuk-link` class is being overridden in such a way that this focus state isn't working? We shouldn't need to have to re-override it...

Comment (phillip-whitaker) [reply src/shared/components/search-result/search-result.component.scss:26]: The specificity of the current styling here is greater than the generic govuk-link class. Therefore the links do style as expected. Which is why I have added this clause here

File: src/shared/components/search-result/search-result.component.scss (hunk 1 above)
Comment (phillip-whitaker) [reply src/shared/components/search-result/search-result.component.scss:9]: This is what I talked about in the tech forum. The colour that comes through on govuk-focus-colour is an 'orangey' yellow and does not match the rest of the website if I do so.

Comment (paul-graham) [reply src/shared/components/search-result/search-result.component.scss:9]: Oh, according to the current GDS style guide, that's the same hex colour. We clearly have an out-of-date one somewhere!

File: src/shared/components/search-result/search-result.component.scss (hunk 2 above)
Comment (paul-graham) [reply src/shared/components/search-result/search-result.component.scss:26]: So I see... but why are we even setting the colour and the background on links within the table? Is that a _different_ blue to the normal link blue? If so, that seems very odd.

**Review by paul-graham — APPROVED** [review]

**Review by adnanakgun — APPROVED** [review]

**Pull request merged**